### PR TITLE
10.2.3 release notes

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # 10.2.x Release Notes
 
+## 10.2.3
+
+This release includes a working preview HTTP/2 client.
```
Comment (johanandren) [inline docs/src/main/paradox/release-notes/10.2.x.md:5]: Not worth it listing individual client issues/prs right?

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+Other notable changes:
+
+#### akka-http-core
+
+* Better support for filename* in Content-Disposition [#3705](https://github.com/akka/akka-http/pull/3705)
```
Comment (ignasi35) [inline docs/src/main/paradox/release-notes/10.2.x.md:11]:
```suggestion
* Better support for filename in Content-Disposition [#3705](https://github.com/akka/akka-http/pull/3705)
```

**Review by ignasi35 — APPROVED** [review]

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
 # 10.2.x Release Notes
 
+## 10.2.3
+
+This release includes a working preview HTTP/2 client.
+
+Other notable changes:
```
Comment (jrudolph) [inline docs/src/main/paradox/release-notes/10.2.x.md:9]:
```suggestion
For now, a connection-level and a "persistent connection" API is provided. The entry points are very similar to the existing flow-based HTTP/1 client APIs (responses are provided out-of-order, though, see the docs). The persistent connection API provides functionality similar to the `newHostConnectionPool` API.

Higher-level APIs for HTTP/2 like the `superPool` or `singleRequest` APIs (or a potential automatic integration into the existing APIs) are still discussed.

Please see the @ref[documentation](/client-side/http2.md) for more information and usage examples.

Other notable changes:
```

**Review by jrudolph — APPROVED** [review]
LGTM (see the suggested change above)

File: docs/src/main/paradox/release-notes/10.2.x.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+
+This release includes a working preview HTTP/2 client. Please consider using this preview to provide early feedback in case you find issues in your use-case. The HTTP/2 client will soon be included as an opt-in backend for the Akka gRPC client.
+
+Native HTTP/2 support in Akka gRPC reduces the number of upstream dependencies and enables more deeply integrated tooling for Akka gRPC in the future.
+
+For now, a connection-level and a "persistent connection" API is provided. The entry points are very similar to the existing flow-based HTTP/1 client APIs (responses are provided out-of-order, though, see the docs). The persistent connection API provides functionality similar to the `newHostConnectionPool` API.
+
+Higher-level APIs for HTTP/2 like the `superPool` or `singleRequest` APIs (or a potential automatic integration into the existing APIs) are still discussed.
+
+Please see the @ref[documentation](/client-side/http2.md) for more information and usage examples.
```
Comment (raboof) [inline docs/src/main/paradox/release-notes/10.2.x.md:13]:
```suggestion
Please see the @ref[documentation](../client-side/http2.md) for more information and usage examples.
```

Comment (jrudolph) [reply docs/src/main/paradox/release-notes/10.2.x.md]: Thanks!

**Review by raboof — APPROVED** [review]

**Pull request merged**